exampleSite/config.toml

```diff
@@ -2,7 +2,7 @@ baseURL = "https://example.com/"
 title = "Indigo Theme"
 copyright = "Copyright © 2018, Author Name"
 languageCode = "en-US"
-theme = "indigo"
+theme = "../.." # Switch back to "indigo"
```

Comment:
Oops, I use this for testing when running `$ hugo server -D` in the `exampleSite` directory; this should have been reverted before committing.

Reply:
Fixed in 502364e